.github/workflows/bash_unit_testing.yml

```diff
@@ -15,6 +14,7 @@ defaults:
 jobs:
   Test-gha-functions:
     name: Tests in ci/gha_functions.sh
+    if: github.repository == 'daos-stack/daos'
```

Comment:
Is this related to pushing PRs from forks? Why would the repo not be `daos-stack/daos`?

Reply:
Kind of, it's related to pushing PRs to forks.  Doing anything in a fork I get lots of email about unable to find any self-hosted runners, with the bash code running on push then even syncing a fork gets me an email with errors in it.

Reply:
That sounds like we still fundamentally don't support PRs from forks then?

Reply:
Not at all.  If there's a PR from a form to daos-stack then this would still trigger.  This change is for if there's a PR to a fork where the action would currently fail due to lack of self-hosted runners.